ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/AbstractBlockProcessor.java

```diff
@@ -46,6 +50,9 @@ TransactionReceipt create(
 
   private static final Logger LOG = LogManager.getLogger();
 
+  private static final Tracer tracer =
+      OpenTelemetry.getGlobalTracer("io.hyperledger.besu.block", "1.0.0");
```

Comment:
`org.hyperledger` instead of `io.hyperledger`?

Reply:
Fixed.